### PR TITLE
fwi-266: Add the tfsec report to CI ci tf config

### PR DESCRIPTION
**Pull request by ivanfetch-fw**
This requires [the accompanying API PR 3291](https://github.com/FairwindsOps/Insights/pull/3291)
## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What's the goal of this PR?
Support the tfsec report in the CI plugin

### CODE REVIEW

**Review by rbren — APPROVED** [review]

File: plugins/ci/pkg/ci/ci.go [hunk 1]
```diff
@@ -402,7 +401,9 @@ func getConfigurationForClonedRepo() (string, string, *models.Configuration, err
 		return "", "", nil, errors.New("IMAGE_VERSION environment variable not set")
 	}
 
-	basePath := filepath.Join("/app", "repository")
+	// marker
+	basePath := filepath.Join("/var/tmp", "repository")
+	// basePath := filepath.Join("/app", "repository")
```
Comment (vitorvezani) [inline plugins/ci/pkg/ci/ci.go:406]: I wonder if you left it commented to revert before merge?

**Review by jdesouza — APPROVED** [review]

File: plugins/ci/pkg/ci/tfsec.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		}
+	}
+	TFSecVersion, err := commands.Exec("tfsec", "-v")
+	if err != nil {
+		return models.ReportInfo{}, fmt.Errorf("cannot get the version of tfsec: %w", err)
+	}
+	TFSecVersion = strings.TrimPrefix(TFSecVersion, "v")
+	report := models.ReportInfo{
+		Report:   "tfsec",
+		Version:  TFSecVersion,
```
Comment (vitorvezani) [inline plugins/ci/pkg/ci/tfsec.go:35]: I wonder if the report version should reflect `tfsec` tool version or `models.TFSecReportProperties` struct changes

File: plugins/ci/pkg/ci/auto_detection_test.go [hunk 3]
```diff
@@ -39,6 +39,9 @@ func TestAutoDetection(t *testing.T) {
 				},
 			},
 		},
+		Terraform: models.TerraformConfig{
```
Comment (vitorvezani) [inline plugins/ci/pkg/ci/auto_detection_test.go:42]: You may want to copy a valid tf file on `./testdata/repo1` to test the auto-detection

File: plugins/ci/pkg/ci/ci.go (hunk 1 above)
Comment (ivanfetch-fw) [reply plugins/ci/pkg/ci/ci.go]: Ooooo yes, I did do that. Thank you!

File: plugins/ci/pkg/ci/tfsec.go (hunk 2 above)
Comment (ivanfetch-fw) [reply plugins/ci/pkg/ci/tfsec.go:35]: I'm currently sending the tfsec version as the report version. We also do this for Trivy. I can see it being useful to vary the report version for internal report changes - is it reasonable to keep the report version aligned with tfsec for now, until we need to vary it?

Comment (vitorvezani) [reply plugins/ci/pkg/ci/tfsec.go:35]: Yes, I think it is reasonable

**Review by vitorvezani — APPROVED** [review]

**Pull request merged**